v2/examples/spiffe-tls/README.md

```diff
@@ -1,73 +1,81 @@
 # Mutually Authenticated TLS (mTLS)
 
-This example shows how two services can communicate using mTLS with X.509 SVIDs obtained from the SPIFFE Workload API. 
+This example shows how to use the go-spiffe library to make two workloads establish an mTLS connection using X.509 SVIDs obtained from the SPIFFE Workload API. 
 
-One of the workloads acts as a client and the other as the server. The use the SPIRE default socket path: `/tmp/agent.sock`. This value can also be set via the `SPIFFE_ENDOINT_SOCKET` environment variable.
+One workload acts as a client and the other as the server. 
 
-The **server workload** creates a listener using the [spiffetls.Listen](https://pkg.go.dev/github.com/spiffe/go-spiffe/v2/spiffetls?tab=doc#Listen) function.
-It uses [spiffetls.MTLSServerWithSourceOptions](https://pkg.go.dev/github.com/spiffe/go-spiffe/v2/spiffetls?tab=doc#MTLSServerWithSourceOptions) to configure  the source, and validates that clients present an X509-SVID with an expected SPIFFE ID.
+The scenario goes like this:
+1. The server starts listening for incoming SPIFFE-compliant mTLS connections.
+2. The client establish an SPIFFE-compliant mTLS connection to the server. 
+3. The server starts waiting a message from the client.
+4. The client sends a "Hello server" message and starts waiting a response.
```

Comment:
waiting a -> waiting for a

Reply:
At leat I wrote it consistently wrong :sweat_smile: